spec/acceptance/management_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper_acceptance'
+
+describe 'rancher::management' do
+  context 'puppet apply' do
+    it 'is expected to apply and be idempotent' do
+      pp = <<-EOS
+class { 'rancher::management':
+  rancher_manager_port => '8080',
+}
+EOS
+
+      apply_manifest(pp, :catch_failures => true)
+      apply_manifest(pp, :catch_changes => true)
+    end
+
+    context 'is expected to be running on the default port' do
+      before :each do
+        sleep 5 # Give it time to start.
```

Comment:
I believe this still won't work. From my test just now, even with a super long wait, it doesn't work :cry: